tests/Integration/MessagingTest.php

```diff
@@ -358,4 +360,54 @@ public function testGetAppInstanceForUnknownToken(): void
         $this->expectException(NotFound::class);
         $this->messaging->getAppInstance(self::$unknownToken);
     }
+
+    /**
+     * @see https://github.com/kreait/firebase-php/issues/713
+     */
+    public function testAndroidConfigTtlWorksWithAPositiveInt(): void
+    {
+        $config = AndroidConfig::fromArray([
+            'ttl' => 1,
+        ]);
+
+        $message = CloudMessage::withTarget(MessageTarget::TOKEN, $this->getTestRegistrationToken())
+            ->withAndroidConfig($config);
+
+        $this->messaging->send($message);
+
+        $this->addToAssertionCount(1);
+    }
+
+    /**
+     * @see https://github.com/kreait/firebase-php/issues/713
+     *
+     * @dataProvider validAndroidConfigTtlValues
+     *
+     * @param int|string $value
+     */
+    public function testAndroidConfigTtlWorksWithValidValues($value): void
+    {
+        $config = AndroidConfig::fromArray([
+            'ttl' => $value,
+        ]);
+
+        $message = CloudMessage::withTarget(MessageTarget::TOKEN, $this->getTestRegistrationToken())
+            ->withAndroidConfig($config);
+
+        $this->messaging->send($message);
+
+        $this->addToAssertionCount(1);
+    }
+
+    /**
+     * @return array<string, list<mixed>>
+     */
+    public function validAndroidConfigTtlValues(): array
+    {
+        return [
+            'int' => [1],
+            'numeric string' => ['1'],
+            'valid string' => ['1s']
```

Comment:
You should add tests for zero strings. Your `int` case checks `> 0` but I'm pretty sure these would pass:
```suggestion
            'valid string' => ['1s'],
            'zero string' => ['0'],
            'zero s string' => ['0s']
```
Not sure if those are desirable.

Reply:
Good catch! I added a1bad48c72e8f64b4482abd07a247df249dc4af0 to account for this, thank you!

PS: I just learned that you can use data providers from other classes (4b2b82c83f564710b7595ca25dea6895b46c45e0)